src/components/TokenCard/TokenCard.tsx

```diff
@@ -143,6 +178,47 @@ export default function TokenCard({ showRedirectModal }: Props) {
     }
   }, [arbitrumIncentiveState?.lp?.isActive, avalancheIncentiveState?.lp?.isActive]);
 
+  const glvsIncentivizedLabel = useMemo(() => {
+    const sparkle = <img src={sparkleIcon} alt="sparkle" className="relative -top-2 -mr-10 inline h-10 align-top" />;
+    const arbitrumLink = <ExternalLink href={getIncentivesV2Url(ARBITRUM)}>Arbitrum</ExternalLink>;
+    const avalancheLink = <ExternalLink href={getIncentivesV2Url(AVALANCHE)}>Avalanche</ExternalLink>;
+
+    const hasArbitrumGlvIncentives =
+      arbitrumIncentiveState?.lp?.isActive &&
+      Object.entries(arbitrumIncentiveState?.lp.rewardsPerMarket ?? {}).some(([market, reward]) => {
+        return Object.values(glvArb ?? {}).some(({ glv }) => glv.glvToken === market && BigInt(reward) > 0n);
```

Comment:
maybe move it to a function `hasSomeRewardForGlv`? very hard to read + repeats for avalanche